test/functional/mempool_accept.py

```diff
@@ -159,64 +139,53 @@ def run_test(self):
         )
 
         self.log.info('A transaction with missing inputs, that existed once in the past')
-        tx = tx_from_hex(raw_tx_0)
-        tx.vin[0].prevout.n = 1  # Set vout to 1, to spend the other outpoint (49 coins) of the in-chain-tx we want to double spend
-        raw_tx_1 = node.signrawtransactionwithwallet(tx.serialize().hex())['hex']
-        txid_1 = node.sendrawtransaction(hexstring=raw_tx_1, maxfeerate=0)
-        # Now spend both to "clearly hide" the outputs, ie. remove the coins from the utxo set by spending them
-        raw_tx_spend_both = node.signrawtransactionwithwallet(node.createrawtransaction(
-            inputs=[
-                {'txid': txid_0, 'vout': 0},
-                {'txid': txid_1, 'vout': 0},
-            ],
-            outputs=[{node.getnewaddress(): 0.1}]
-        ))['hex']
-        txid_spend_both = node.sendrawtransaction(hexstring=raw_tx_spend_both, maxfeerate=0)
+        node.sendrawtransaction(hexstring=tx_0['hex'], maxfeerate=0) # spend the tx_0
+        self.mempool_size += 1
+        tx_1 = miniwallet.send_self_transfer(fee_rate=4*fee_rate,from_node=node)
+        utxo_1 = miniwallet.get_utxo()
+        miniwallet.send_self_transfer(from_node=node, utxo_to_spend=utxo_1)['txid']
```

Comment:
```suggestion
        miniwallet.send_self_transfer(from_node=node)['txid']
```

should be identical

Reply:
Also, why the `['txid']` in the end?

Reply:
Removed the unnecessary `['txid']`. The suggestion is not identical, since `miniwallet.send_self_transfer` sorts the utxo set by the value, before choosing a transaction.